Dangerfile.swift

```diff
@@ -361,7 +362,7 @@ extension DangerDSL {
         // The volume of diff should not be over 1,000 lines.
         let doDiffAmountCheckTitle = "Diff Volume Check"
         result.check(doDiffAmountCheckTitle) {
```

Comment:
<!--
  0 failure: 
  1 warning:  Underscores shoul...
  
  
  DangerID: danger-id-Danger;
  File: Dangerfile.swift;
  Line: 364;
-->

- :warning: Underscores should be used as thousand separator in large decimal numbers. (`number_separator`)